From a3bd5917e96735c4cd2633f610e263acc83efd25 Mon Sep 17 00:00:00 2001
From: Jiping Ma <jiping.ma2@windriver.com>
Date: Mon, 11 Apr 2022 02:07:30 -0700
Subject: [PATCH] mlnx-tools: Fix the build error.

This changes 5.2.0 from 5.2.0-OFED.5.5.1.0.3 in changelog to
fix the following build error.

debrepack - ERROR - dpkg-architecture: warning: cannot determine CC system type, falling back to default (native compilation)
debrepack - ERROR - dpkg-architecture: warning: cannot determine CC system type, falling back to default (native compilation)
debrepack - ERROR -  dpkg-source --before-build .
debrepack - ERROR -  dpkg-source --before-build .
debrepack - ERROR - dpkg-buildpackage: warning: building a source package without cleaning up as you asked; it might contain undesired files
debrepack - ERROR - dpkg-buildpackage: warning: building a source package without cleaning up as you asked; it might contain undesired files
debrepack - ERROR -  dpkg-source -b .
debrepack - ERROR -  dpkg-source -b .
debrepack - ERROR - dpkg-source: error: can't build with source format '3.0 (native)': native package version may not have a revision
debrepack - ERROR - dpkg-source: error: can't build with source format '3.0 (native)': native package version may not have a revision
debrepack - ERROR - dpkg-buildpackage: error: dpkg-source -b . subprocess returned exit status 25
debrepack - ERROR - dpkg-buildpackage: error: dpkg-source -b . subprocess returned exit status 25
debrepack - ERROR - [ Failed - "cd /localdisk/loadbuild/jma1/stx/std/mlnx-tools/mlnx-tools-5.2.0; dpkg-buildpackage -nc -us -uc -S -d" ]

Signed-off-by: Jiping Ma <jiping.ma2@windriver.com>
---
 debian/changelog | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/changelog b/debian/changelog
index 0a1dc57..c9b4370 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,4 +1,4 @@
-mlnx-tools (5.2.0-OFED.5.8.1.0.0) UNRELEASED; urgency=medium
+mlnx-tools (5.2.0) UNRELEASED; urgency=medium
 
   * Switched to a MLNX_OFED branch
   * Native package
-- 
2.34.1

